Userland/Libraries/LibWeb/DOM/EventDispatcher.cpp

```diff
@@ -89,12 +89,9 @@ bool EventDispatcher::inner_invoke(Event& event, Vector<EventTarget::EventListen
 
         auto* this_value = Bindings::wrap(global, *event.current_target());
         auto* wrapped_event = Bindings::wrap(global, event);
-        auto& vm = global.vm();
-        [[maybe_unused]] auto rc = JS::call(global, function, this_value, wrapped_event);
-        if (vm.exception()) {
-            vm.clear_exception();
-            // FIXME: Set legacyOutputDidListenersThrowFlag if given. (Only used by IndexedDB currently)
-        }
+        (void)JS::call(global, function, this_value, wrapped_event);
+        // FIXME: Do we need to show unhandled exceptions here?
```

Comment:
See the spec attached at the start of the function. The second comment is now meaningless, please make clear that it is a substep of step 10 ("If this throws an exception, then:")